application.go

```diff
@@ -8,14 +8,15 @@ import (
 	"github.com/gookit/color"
 	"github.com/pkg/profile"
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 	"github.com/wagoodman/go-partybus"
 	"gopkg.in/yaml.v3"
 
 	"github.com/anchore/fangs"
 	"github.com/anchore/go-logger"
 )
 
-type Initializer func(cfg Config, state State) error
+type Initializer func(cfg *Config, state State) error
```

Comment:
same comment about immutability of the config https://github.com/anchore/clio/pull/5/files#r1205715970